### ISSUE TITLE
Question: How is FreeCAD involved in this project?

### ISSUE THREAD

**Issue by luzpaz** [Dessia-tech/volmdlr#1]
Looks interesting. Would you mind elaborating ?

**Comment by masfaraud** [comment]
Hi,
The aim of the project was to have a simple geometric solver for 2D problems in python (computing areas, inertia...), defining rounded contours...
Based on this 2D description, 3D primitives are build using a FreeCAD binding (cf FreeCADExport in core.VolumeModel). In the future, we will try to have our own geometrical description with NURBS and so on.
Our binding enables to use FreeCAD in python3. To perform the FreeCADExport, the user has to give the python path which can import FreeCAD.
I think our best test case is the casing:
https://github.com/Dessia-tech/volmdlr/blob/master/scripts/casing.py
What do you think?

**Comment by luzpaz** [comment]
Looks very cool!
Would you be interested in posting this to the FreeCAD community on the [Users Showcase subforum](https://forum.freecadweb.org/viewforum.php?f=24) ? I'm sure there are people that would also appreciate this work. Thanks!

**Comment by masfaraud** [comment]
Yes, absolutely! There is still a little work to do before that like a gallery of examples and a doc.
We keep in touch!

**Comment by luzpaz** [comment]
Any progress ?

**Comment by masfaraud** [comment]
Yes, heavy dev in progress on branch step: reading/writing step, support of Bsplines...
